tests/integration/MANIFEST

```diff
@@ -1,39 +0,0 @@
-# file GENERATED by distutils, do NOT edit
```

Comment:
we should add this file to gitignore

Reply:
yes, this is being added by another PR open